core/fetcher/fetcher.go

```diff
@@ -196,6 +199,11 @@ func (f *Fetcher) fetchAggregatorData(ctx context.Context, slot int64, defSet co
 		}
 		log.Info(ctx, "Resolved attester aggregation duty", z.Any("pubkey", pubkey))
 
+		_, ok = aggAttByCommIdx[attDef.CommitteeIndex]
+		if ok {
+			continue // Skips querying aggregate attestation for aggregators of same committee.
```

Comment:
please follow `fetchAttesterData` example, this will result in `resp[pubkey]` not being set for this validator

Reply:
please also add a test for this edge-case

Reply:
Done!
Setting response for this validator also added tests for this case, converted existing tests to table tests.